backend/drone/drone.js

```diff
@@ -0,0 +1,25 @@
+const axios = require("axios");
+
```

Comment:
add comment about the host (0.0.0.0 when running both backend and frontend on the same machine)